src/coffee/constants.coffee

```diff
@@ -81,4 +81,31 @@ exports.SPECIAL_HEADERS = [
   # TODO: image labels and dimensions
 ]
 
+exports.PRODUCT_LEVEL_PROPERTIES = [
+  constants.HEADER_ID,
+  constants.HEADER_KEY,
+  constants.HEADER_PRODUCT_TYPE,
+  constants.HEADER_NAME,
+  constants.HEADER_DESCRIPTION,
+  constants.HEADER_CATEGORY_ORDER_HINTS,
+  constants.HEADER_SLUG,
+  constants.HEADER_CATEGORIES
+  constants.HEADER_META_TITLE
+  constants.HEADER_META_DESCRIPTION
+  constants.HEADER_META_KEYWORDS
+  constants.HEADER_SEARCH_KEYWORDS
+  constants.HEADER_STATE
+  constants.HEADER_SEARCH_KEYWORDS
+  'version',
+  'createdAt',
+  'lastModifiedAt',
+  'categories',
+  'hasStagedChanges',
+  'published',
+  'masterVariant',
+  'variants',
+  'taxCategory',
+  'reviewRatingStatistics'
+]
```

Comment:
LGTM, could we please also add that link to where to find these? Just to make sure it's easy to keep everything up-to-date if/when needed.

Reply:
Hello @daern91 .. I added a comment here a0782a2